action/action_deserializer.go

```diff
@@ -9,12 +9,25 @@ package action
 import "github.com/iotexproject/iotex-proto/golang/iotextypes"
 
 // Deserializer de-serializes an action
+//
+// It's a wrapper to set certain parameters in order to correctly de-serialize an action
+// Currently the parameter is EVM network ID for tx in web3 format, it is called like
+//
+// act, err := (&Deserializer{}).SetEvmNetworkID(id).ActionToSealedEnvelope(pbAction)
+//
```

Comment:
explanatory comments per haixiang's earlier comment

Reply:
a possibly better solution might be:
```
act, err := action.NewSerializer(ctx).ActionToSealedEnvelope() // NewSerializer will extract evmNetworkID from ctx
```
so we don't need to explicitly call `SetEvmNetworkID(id)` everywhere

however this is impossible now due to import cycle, moving `context.go` from `/action/protocol` to `/action` could unblock this, maybe a further task